redisbp/hooks.go

```diff
@@ -49,7 +48,10 @@ func (h SpanHook) AfterProcessPipeline(ctx context.Context, cmds []redis.Cmder)
 			errs.Add(cmd.Err())
 		}
 	}
-	return h.endChildSpan(ctx, errs.Compile())
+	_ = h.endChildSpan(ctx, errs.Compile())
```

Comment:
nit: you can remove the `_ =` part.

Reply:
Also I think at this point there's no reason for `endChildSpan` to return an error, so that function can be changed to return nothing.

Reply:
Ya, I think `endChildSpan` should return nothing and we just return `nil` for the error in each `After` function?

Reply:
Yeah, I also changed another part where it was used.